### DISCUSSION TITLE
JavaScript Code Sample 2

### DISCUSSION THREAD

**Discussion by pdp2121** [XRPLBounties/Proposals#77] [Open Bounties]
### Discussion thread for bounty: [JavaScript Code Sample 2](https://github.com/XRPLBounties/Proposals/pull/75)
Add more code samples, and fix depreciated ones to capture more common tasks from users.

**Comment by jonathanlei** [comment]
This [Bounty](https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0077%20JavaScript%20Code%20Samples%202.md) is now open! Please submit your work on xrpl.org repo

**Comment by AlexanderBuzz** [comment]
I am working on 5 ([Build a Wallet](https://github.com/XRPLF/xrpl-dev-portal/tree/ec25f6bbddf182550186406f85913014aa423f44/content/_code-samples/build-a-wallet)), this will include the code as well as documentation that can be put on xrpl.org aka "Build a Desktop Wallet in JavaScript"

**Reply by AlexanderBuzz** [reply]
So while I'm at it, I'll give a shot at 3 (Airgapped Wallet) as well.

**Comment by TusharPardhe** [comment]
I've picked up 1, 2 & 6 :)

**Comment by JST5000** [comment]
We've accepted @TusharPardhe's contributions for Milestones 1 and 2
* https://github.com/XRPLF/xrpl-dev-portal/pull/1705
* https://github.com/XRPLF/xrpl-dev-portal/pull/1700

And we've accepted @wojake's contributions for Milestones 7, 8, 9, and 10
* https://github.com/XRPLF/xrpl-dev-portal/pull/1689
* https://github.com/XRPLF/xrpl-dev-portal/pull/1684
* https://github.com/XRPLF/xrpl-dev-portal/pull/1682
* https://github.com/XRPLF/xrpl-dev-portal/pull/1685

Please submit to claim those bounties here if you haven't already :) : [https://xrpl.typeform.com/xrpl-bounties](https://xrpl.typeform.com/xrpl-bounties?utm_source=xxxxx&utm_medium=xxxxx&utm_campaign=xxxxx&utm_term=xxxxx&utm_content=xxxxx)

**Reply by JST5000** [reply]
Edited to include @wojake's submission for milestone 9 since I missed that in my original comment - sorry about that!

**Reply by wojake** [reply]
https://github.com/XRPLF/xrpl-dev-portal/pull/1687: This PR has accepted work

**Comment by JST5000** [comment]
We've accepted @AlexanderBuzz's contribution for Milestone 3 :)
* https://github.com/XRPLF/xrpl-dev-portal/pull/1693
